processor/k8sattributesprocessor/README.md

```diff
@@ -332,3 +332,26 @@ as tags.
 
 By default, the `k8s.pod.start_time` uses [Time.MarshalText()](https://pkg.go.dev/time#Time.MarshalText) to format the
 timestamp value as an RFC3339 compliant timestamp.
+
+## Feature Gate
+
+### `k8sattr.fieldExtractConfigRegex.disallow`
```

Comment:
Can you provide some information here on how to migrate to `ExtractPatterns` in the transform processor? What you have in the changelog looks like a pretty good place to start.